src/main/java/com/redhat/engineering/srcclr/processor/SecurityDataProcessorResult.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.engineering.srcclr.processor;
+
+import com.redhat.engineering.srcclr.json.securitydata.SecurityDataJSON;
+
+public class SecurityDataProcessorResult {
+    private Boolean to_notify = false;
+    private Boolean to_fail = false;
+    private String message = "";
+    private SecurityDataJSON json = null;
+
```

Comment:
Given lombok is already being used can we used @Getter/@Setter on this ?  ( See https://projectlombok.org/features/GetterSetter )

Reply:
ok. will try.